### PR TITLE
Updated pagination limit for order items query 

### PR DESCRIPTION
**Pull request by lgalis**
Updated pagination limit for order items query to account for the possible pre/post.. 

Fixes https://issues.redhat.com/browse/SSP-2065

When retrieving the order items for a page of order, some of the items were not retrieved due to pagination limit.

### CODE REVIEW

File: src/smart-components/common/query_log [hunk 1]
```diff
@@ -0,0 +1,102 @@
...%5D%5B%5D%5B%5D=9322\u0026filter%5Border_id%5D%5B%5D%5B%5D=9320\u0026filter%5Border_id%5D%5B%5D%5B%5D=9256\u0026filter%5Border_id%5D%5B%5D%5B%5D=9187\u0026filter%5Border_id%5D%5B%5D%5B%5D=9186\u0026filter%5Border_id%5D%5B%5D%5B%5D=9185\u0026filter%5Border_id%5D%5B%5D%5B%5D=9184\u0026filter%5Border_id%5D%5B%5D%5B%5D=9183\u0026filter%5Border_id%5D%5B%5D%5B%5D=9182\u0026filter%5Border_id%5D%5B%5D%5B%5D=9181\u0026filter%5Border_id%5D%5B%5D%5B%5D=9180\u0026filter%5Border_id%5D%5B%5D%5B%5D=9178\u0026filter%5Border_id%5D%5B%5D%5B%5D=9176\u0026filter%5Border_id%5D%5B%5D%5B%5D=8854\u0026filter%5Border_id%5D%5B%5D%5B%5D=8853\u0026filter%5Border_id%5D%5B%5D%5B%5D=8852\u0026filter%5Border_id%5D%5B%5D%5B%5D=8851\u0026filter%5Border_id%5D%5B%5D%5B%5D=8850\u0026filter%5Border_id%5D%5B%5D%5B%5D=8849\u0026filter%5Border_id%5D%5B%5D%5B%5D=8848\u0026filter%5Border_id%5D%5B%5D%5B%5D=8383\u0026filter%5Border_id%5D%5B%5D%5B%5D=8381\u0026filter%5Border_id%5D%5B%5D%5B%5D=8380\u0026filter%5Border_id%5D%5B%5D%5B%5D=8378\u0026filter%5Border_id%5D%5B%5D%5B%5D=8307\u0026filter%5Border_id%5D%5B%5D%5B%5D=8306\u0026filter%5Border_id%5D%5B%5D%5B%5D=8305\u0026filter%5Border_id%5D%5B%5D%5B%5D=8304\u0026offset=100"},
```
Comment (Hyperkid123) [inline src/smart-components/common/query_log:1]: Is this file required?

Comment (lgalis) [reply src/smart-components/common/query_log:1]: no, thanks for noticing

**Review by Hyperkid123 — APPROVED** [review]
